src/client/src/containers/AzureSubscriptions/index.tsx

```diff
@@ -235,6 +235,15 @@ class AzureSubscriptions extends React.Component<Props, IState> {
     const { isLoggedIn, setDetailPage, isPreview } = this.props;
     const serviceTypes = azureServiceOptions.map(option => option.type);
     const uniqueServiceTypes = [...new Set(serviceTypes)];
+
+    let numHostingServiceCards = 0;
+    azureServiceOptions.forEach(serviceOption => {
+      const isCardShown = isPreview || !serviceOption.isPreview;
+      if (serviceOption.type === servicesEnum.HOSTING && isCardShown) {
+        numHostingServiceCards++;
+      }
+    });
+
```

Comment:
wait. what is this..

Reply:
this is logic so that the "You can only choose one hosting service" message only shows up if there's more than 1 hosting services card

Reply:
also for preview mode too (it will have 2 hosting services cards)